### ISSUE TITLE
Reach Entity Attributes lib breaks chest animations & sound

### ISSUE THREAD

**Issue by Abalieno** [sf-inc/sihywtcamc#34]
I verified this with this small mod list (1.19.2/fabric):

![tymjtyj](https://github.com/sf-inc/sihywtcamc/assets/4656436/8ae9f084-ca07-4071-a4a6-ca44b6f950fd)

Tried to set "reachAttribute" to false, but it didn't fix anything.

The problem, easy to reproduce, is that when you interact with a double chest right at the max range possible, the chest gets permanently bugged until a reload.

The chest works, in the sense that you can take or drop items, but animations and sound get bugged.

It usually happens one of the three, or a mix of them:
- Only one side opens.
- No sound.
- No animation.

Most of the times both sound and animations stop.

When this bug happens once on a chest, that chest gets stuck permanently in that state. So if the animation stops once, then it will never play again, even if you interact with the chest at close range.

It doesn't seem to happen with single chests.

![tyjtyj](https://github.com/sf-inc/sihywtcamc/assets/4656436/ac5afa14-e114-4dbc-9863-d1bf1249f597)

**Comment by Abalieno** [comment]
Can't I get some kind of answer?

**Comment by charlyb01** [comment]
Hello and thanks for the report! I'll test and try to reproduce. I'll get back to you by next week

**Comment by charlyb01** [comment]
I have been able to reproduce and here are a few things I noticed:
* This can also happen with single chests
* This is due to the Reach Entity Attributes library
* In other mods playing with inventories, the same kind of bugs can be noticed, so screen handling might be tricky

I didn't find what was the cause sadly, but I'll post something on your issue later. Also, note that I'm going to remove attributes changes (and all other combat snapshot features) in the next update. So, I'm going to close this issue at this moment.
